csharp/test/Ice/proxy/AllTests.cs

```diff
@@ -597,8 +597,15 @@ public static IMyClassPrx Run(TestHelper helper)
             property = propertyPrefix + ".Router";
             TestHelper.Assert(b1.Router == null);
             communicator.SetProperty(property, "ice+tcp://host:10000/router");
-            b1 = communicator.GetPropertyAsProxy(propertyPrefix, IObjectPrx.Factory)!;
-            TestHelper.Assert(b1.Router != null && b1.Router.Identity.Name == "router");
+            try
+            {
+                b1 = communicator.GetPropertyAsProxy(propertyPrefix, IObjectPrx.Factory)!;
```

Comment:
Maybe add a comment here that this fails because the router must be an ice1 proxy

Reply:
Fixed.